lib/Clam/CfgBuilder.cc

```diff
@@ -1833,16 +1833,23 @@ void CrabIntraBlockBuilder::visitSelectInst(SelectInst &I) {
     assert(cond->isVar());
 
     if (op1->isVar() && op2->isVar()) {
-      m_bb.select_ref(lhs->getVar(), lhs_rgn, cond->getVar(), op1->getVar(),
-                      op1_rgn, op2->getVar(), op2_rgn);
+      opt_pair_var_t pair_op1 = std::make_pair(op1->getVar(), op1_rgn);
```

Comment:
Factorize code to have a single call to `insertCrabIRWithEmitter::select_ref`.
That is, 

```
opt_pair_var_t pair_op1 = llvm::None;
opt_pair_var_t pair_op2 = llvm::None;
if (op1->isVar() && op2->isVar()) {
     pair_op1 = ...
     pair_op2 = ...
} else if (!op1->isVar()) {
    pair_op2 = ...
} else if (!op2->isVar()) {
    pair_op1 = ...
}

if (pair_op1.hasValue() || pair_op2.hasValue()) {
    insertCrabIRWithEmitter::select_ref(....);
} 
```